vagrant/lib/vagrant-zeus/commands/zeus-file-monitor.rb

```diff
@@ -34,15 +34,15 @@ def run
                 begin
                   modified_files_buf += @file_monitor.readpartial(4096)
                 rescue EOFError
-                  puts "lost connection to the file monitor process, exiting!"
+                  puts "[#{Time.now.strftime('%H:%M:%S')}] lost connection to the file monitor process, exiting!"
```

Comment:
@shalecraig: can you move this prefixing into a private `log` helper on `FileWatcher`? Then I'll get this merged!